### PR TITLE
feat: Update meme template IDs name

### PR DESCRIPTION
**Pull request by anddreluis2**
- Update templateId in imgflipCaption function calls to match the new template IDs for different memes.
- Replace randomTemplatedId with templateId in the function arguments for consistency.
- This change ensures that the correct meme templates are used when generating captions for memes.

### CODE REVIEW

File: src/imgflip/imgflipCaption.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { imgflipApi } from './imgflipApi';
 import { objToUrlEncoded } from '../utils/objToUrlEncoded';
 
 type IImgflipCaptionArgs = {
-  randomTemplatedId: string;
+  templateId: string;
```
Comment (vinibgoulart) [inline src/imgflip/imgflipCaption.ts:6]: ty!

File: src/wweb/onMessage/meme/supremacyMessage.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import { Message, MessageMedia } from 'whatsapp-web.js';
+import { imgflipCaption } from '../../../imgflip/imgflipCaption';
+import imageToBase64 from 'image-to-base64';
+import { middleware } from '../../middleware/middleware';
+
+const supremacyMessage = async (msg: Message) => {
```
Comment (vinibgoulart) [inline src/wweb/onMessage/meme/supremacyMessage.ts:6]: could we add this in the `commands.ts`?

**Review by vinibgoulart — APPROVED** [review]
nice memes!

File: src/wweb/onMessage/meme/supremacyMessage.ts (hunk 2 above)
Comment (anddreluis2) [reply src/wweb/onMessage/meme/supremacyMessage.ts:6]: done

**Pull request merged**